bitcoin/src/pow.rs

```diff
@@ -924,9 +926,137 @@ impl kani::Arbitrary for U256 {
     }
 }
 
+/// Gets the target for the block after `current_header`.
+///
+/// Implements the `GetNextWorkRequired` function from Bitcoin core.
```

Comment:
It is better to add the link of this function from a commit hash, in this way
if the core function goes away in the future we know where to look

Reply:
Ah good idea, thanks!